diagnostic.md

```diff
@@ -21,13 +21,21 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include -request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+const getDonuts = function (form) {
+  let data = getFormFields(form);
```

Comment:
why a form here and why use "data" if you don't use it anywhere in this function?